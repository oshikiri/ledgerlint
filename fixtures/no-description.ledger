2020/04/02
  Expences:Household essentials  200 JPY
  Assets:Cash

2020/04/02 *
  Expences:Household essentials  200 JPY
  Assets:Cash

2020/04/02 * ; some comment
  Expences:Household essentials  200 JPY
  Assets:Cash
