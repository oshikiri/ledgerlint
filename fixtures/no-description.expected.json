{"type":"decoration","source":"ledgerlint","file_path":"fixtures/no-description.ledger","range":{"start":{"line":2,"character":0},"end":{"line":2,"character":80}},"renderOptions":{"after":{"contentText":"- 200 JPY","color":"grey","margin":"2ch"}}}
{"type":"decoration","source":"ledgerlint","file_path":"fixtures/no-description.ledger","range":{"start":{"line":6,"character":0},"end":{"line":6,"character":80}},"renderOptions":{"after":{"contentText":"- 200 JPY","color":"grey","margin":"2ch"}}}
{"type":"decoration","source":"ledgerlint","file_path":"fixtures/no-description.ledger","range":{"start":{"line":10,"character":0},"end":{"line":10,"character":80}},"renderOptions":{"after":{"contentText":"- 200 JPY","color":"grey","margin":"2ch"}}}
{"type":"decoration","source":"ledgerlint","file_path":"fixtures/no-description.ledger","range":{"start":{"line":15,"character":0},"end":{"line":15,"character":80}},"renderOptions":{"after":{"contentText":"- 200 JPY","color":"grey","margin":"2ch"}}}
